src/wallet/wallet.cpp

```diff
@@ -2093,7 +2093,7 @@ int CWallet::ScanForWalletTransactions(CBlockIndex* pindexStart, bool fUpdate)
         pcoinsTip->GetScIds(allScIds);
         for(const auto& scId: allScIds)
         {
-            if (pcoinsTip->isCeasedAtHeight(scId, chainActive.Height()) != CSidechain::State::ALIVE) {
```

Comment:
Do we need `isCeasedAtHeight` method at all? Seems we always need to use `GetSidechainState`

Reply:
No, you are right. It is only in UT for the time being we chose to leave it there, but we might as well remove it.

Reply:
Added a comment in the header for that